scope.md

```diff
@@ -91,6 +91,51 @@ priority:
 * QR-code triggered payment. User points phone at QR code which represents a Web Page. In vanilla mode, Web Page includes "Buy" button to trigger Payment Request. But in streamlined mode, SPC transaction dialog is immediately displayed to the user for a default instrument associated with the payment method.
 * Tap-to-pay (NFC)
 
+## User Stories
+
+### Out of band enrollment
+
+* While mobile banking, Alice enroll her authenticator to be used with all her credit cards issued by this bank.
```

Comment:
I <3 this as it captures the 'enroll all my payment instruments' use-case that the SPC Origin Trial doesn't currently enable well (and which I think is an important use-case). I'd love for the task force to chat about this and get some opinions in the open :)

Reply:
Hearing the "+1" can we resolve this comment?